config/swagger.yaml

```diff
@@ -183,12 +180,7 @@ paths:
       produces:
         - application/json
       parameters:
-        - in: path
-          name: height
-          description: 'The height of the block'
-          required: true
-          type : integer
-          minimum: 0
+        - $ref: '#/parameters/HeightIn'
```

Comment:
The path object is defined [here](https://github.com/OAI/OpenAPI-Specification/blob/8a387ac71f7f31aaff5b910f1e0e99ad12ff3ef0/versions/2.0.md#pathItemObject).

`parameters` is defined as `[Parameter Object | Reference Object]`. So this can be a ref. Ok.